source/_components/binary_sensor.threshold.markdown

```diff
@@ -9,7 +9,7 @@ sharing: true
 footer: true
 logo: home-assistant.png
 ha_category: Binary Sensor
-ha_iot_class: "Local Polling"
+ha_iot_class: "Local Push"
```

Comment:
I find the whole IoT Class a weird thing to have on the "meta-sensors" in general. But in this case it's _listening_ to the sensor that it is monitoring, which is basically a push. This will react instantly to state changes rather than having to wait for a polling timer.

I feel like a new "Internal" IoT class is most appropriate, but that's a separate discussion. Perhaps the IoT class should be removed from this sensor entirely?